scripts/ci-gcc-11-Fortran-double-complex.sh

```diff
@@ -4,9 +4,9 @@ set -e -u -x
 
 basedir=$(readlink --canonicalize $(dirname $0)/..)
 
-export CC=${CC:-gcc-11}
-export CXX=${CXX:-g++-11}
-export FC=${FC:-gfortran-11}
+export CC=${CC:-gcc-4.8}
```

Comment:
why is it 4.8 when 11 is in the filename?

Reply:
Thanks! I fixed it.